src/store/toast.store.ts

```diff
@@ -0,0 +1,97 @@
+type ToastMessage = {
+  type: 'success' | 'error'
+  message: string
+  id: number
+  show: boolean
```

Comment:
You don't need to manually maintain a state variable for showing the toast message, or add timeouts for showing and hiding the message (`showToast`, `hideToast`). A toast is shown if you add it to the message array, and removed if you remove it from the array. The `Transition` group on each toast message will take care of animating on enter and leave.

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/98/commits/095227ffdb9fb9701afe1274203c968090cdb85b